docs/UserGuide.md

```diff
@@ -188,16 +189,16 @@ Examples:
 
 Deletes the specified persons from the address book.
 
-Format: `delete [INDEX | INDEX-INDEX]…`
+Format: `delete INDEX | LowerBound-UpperBound [INDEX]… [LowerBound-UpperBound]…`
```

Comment:
Is making one compulsory field necessary? It makes the command format look long and cluttered.

Additionally, `edit` command has a similar situation, which was solved by adding a remark stating that at least one optional field is required.